olp-cpp-sdk-core/include/olp/core/thread/Continuation.h

```diff
@@ -46,7 +45,7 @@ class CORE_API ContinuationImpl final {
  public:
   /// An alias of a function which returns an error as a callback.
   using FailedCallback = std::function<void(client::ApiError)>;
-  /// Type of return value of Continuation Task.
+  /// Type of return value of Continuation task.
```

Comment:
The return value type of the `Continuation` task.

Reply:
done